tests/acceptance/click-view-profile-test.js

```diff
@@ -8,7 +8,7 @@ module('Acceptance | click view profile', function(hooks) {
   test('visiting /', async function(assert) {
     await visit('/');
     await click('.view-profile-button');
-    await waitFor('h3#sex-and-age');
+    await waitFor('h3#sex-and-age', { timeout: 2000 });
```

Comment:
Hmmm... relying on timeouts can lead to flakey tests.

What could be happening is that somewhere a Promise isn't being [tracked](https://emberjs.com/api/ember/3.7/functions/@ember%2Ftest/registerWaiter) for some reason.

This is fine for now but I would try to avoid in the future when possible.

Reply:
The default timeout for waitFor is 1000 , so we're implicitly using timeouts. This takes longer b/c my new backend is slow :( working on optimizing and will clean this up then.